### DISCUSSION TITLE
Payment declined; next try in 5 days; copilot not working; no option to “pay now”

### DISCUSSION THREAD

**Discussion by vzakharov** [community/community#30884] [New to GitHub]
Hi, Github couldn’t charge my card today (insufficient funds). I topped it up, but the billing page says it will retry on Aug 31 (5 days from now). 

In the meantime, my copilot subscription became inactive, and I don’t see a way to charge my card manually for it right now.

Should I really wait for 5 days to get copilot back up and running, or am I missing something?

**Comment by vzakharov** [comment]
Update: ironically, it started working after I canceled copilot subscription (because it is still active until Sep 25). I just hope I’ll be able to re-subscribe.

**Reply by orenelbaum** [reply]
Were you able to re-subscribe?

**Reply by vzakharov** [reply]
yep but I don't remember how 🙈

**Comment by MaxOlsson** [comment]
I have the same problem. I had no money on my card when it tried to charge me and when I added funds to my card I found no way to have Co-pilot retry my card. Should I also have to wait 5 days?

It would be nice with a way to force Github/Copilot to retry the card

**Reply by BigDlamz** [reply]
This is such a silly problem to have, retrying a failed payment literally brings in revenue for them. 

Their payments page is too complicated. It should have one button, retry payment, thats it.

**Reply by AsifIqbal-debug** [reply]
Agreed. There should be a Retry button.

**Comment by joshdev15** [comment]
In my case, my bank has a "padlock" that I must disable to make a payment, which is convenient for me because I am protected from unauthorized charges and other similar... the solution I think would be good to add a button to retry the payment on the "settings" page > "payment management" in this way to be able to make the payment at your convenience. However, what I did was cancel my subscription and then subscribe again, it seems to me a better solution than waiting 5 days (which I will eventually forget) to disable the lock again... :(

**Comment by yusronhanan** [comment]
This is an important features to be addressed, IMO

**Reply by miker720** [reply]
this is embarrassing.. how can it not have a way to manually submit.

**Comment by tommyopeters** [comment]
I'm having the exact same issue, except it seems that I'd have to wait 6 days. A workaround I found is to just resubscribe directly for Github Copilot. Attempt to access the Free Trial and it'll show you a page where you'd have to pay.

**Comment by geexup** [comment]
You can delete your payment information and then re-enter the same card details. This action will initiate a payment process, allowing you to purchase Copilot.

**Reply by linuszetterman-C1** [reply]
Worked like a dream. Thanks!

**Reply by renuka-fernando** [reply]
This worked, Thanks!

**Reply by anon11412** [reply]
worked

**Reply by ahid-technologies** [reply]
Thanks!

**Reply by DevIsraelSantos** [reply]
Fev/2026... This worked, Thanks!

**Reply by PWrzesinski** [reply]
Didn't work for me

**Comment by cbruguera** [comment]
You need a "retry payment" button. It's not that complicated. 🙏🏼

**Comment by rexson98** [comment]
This is still an issue. 4 years ago but no retry payment button yet.
